tests/integration/test_first_seen_correction_integration.py

```
@@ -0,0 +1,452 @@
+
+
+pytestmark = [pytest.mark.integration]
+
+
+FIXTURE_PATH = (
+    Path(__file__).resolve().parent.parent.parent
+    / 'docs' / 'development' / 'example-data' / '1aeo_relays_data.json'
+)
+
+
+def _load_fixture():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_load_fixture`

(ANN202)